just/dfx.just

```diff
@@ -124,10 +131,27 @@ create_canisters network cycles:
 
 
 [private]
-dfx_local_deploy_token_storage owner:
+dfx_local_deploy_token_storage owner network="":
   #!/usr/bin/env bash
   set -e
 
+  # If network is empty (local), include the test tICP token entry. Otherwise leave empty.
```

Comment:
This is a very weak condition. You should not rely on it. What happens if the caller use `local` instead of enpty? What if add new environments that need this data?